blueman/main/Sendto.py

```diff
@@ -137,10 +139,10 @@ def __init__(self, device, adapter_path, files):
 
         self.show()
 
-    def create_session(self):
+    def create_session(self) -> None:
         self.client.create_session(self.device['Address'], self.adapter["Address"])
 
-    def on_cancel(self, button):
+    def on_cancel(self, button: Optional[Gtk.Button]) -> None:
```

Comment:
button should never be None', https://lazka.github.io/pgi-docs/#Gtk-3.0/classes/Button.html#Gtk.Button.signals.clicked. I see we check for the button but I don't understand why.

Reply:
`on_response` in `on_transfer_error` calls this directly with button set to `None`.

Reply:
Did not think we did that, ok then.